oxidation/client/src/transitions/SlideHorizontal.vue

```diff
@@ -15,12 +15,12 @@ const props = defineProps<{
 
 <style>
 .slide-left-enter-active, .slide-right-enter-active {
-  transition: 0.5s ease-in-out;
-  transition-delay: 0.5s;
+  transition: .4s ease-out;
+  transition-delay: 0.3s;
```

Comment:
should stay the same as 'transition', to avoid possible bugs (because of how <Transition> works with Vue)